### ISSUE TITLE
always allow to enable MVT layer

### ISSUE THREAD

**Issue by karussell** [graphhopper/graphhopper#2129]
Currently the layer cannot be selected for low zoom levels.

**Comment by easbar** [comment]
Yeah at least there should be some hint to indicate that it is disabled when we the map is zoomed out. Right now clicking the 'Local MVT' checkbox at a low zoom level does nothing and it just seems something is broken.

**Comment by karussell** [comment]
This is an inbuilt behaviour of Leaflet I think. Nothing under our control (I think). So we can only just allow all zoom levels for MVT and have to make sure that performance is fine for low zoom levels.

**Comment by boldtrn** [comment]
Something that you probably could do is to update the label of the MVT layer on a zoom event, so you could change the label like (zoom in 2 more steps).

**Comment by easbar** [comment]
Ah ok, we could also change the text from 'Local MVT' to 'MVT (high zoom only)'... Don't know :). But I'd be surprised if its not possible to add some clickhandler to the checkbox and show some alert or something

**Comment by easbar** [comment]
Just for reference: The minZoom level is currently set to 10 here: https://github.com/graphhopper/graphhopper/blob/a17b6d5d23af880c8cb5447ef53e61ec4d7af08e/web/src/main/resources/assets/js/config/tileLayers.js#L105
